New package: EpicHyperSketch v0.0.1

JuliaRegistrator opened the pull request:
- Registering package: EpicHyperSketch
- Repository: https://github.com/kchu25/EpicHyperSketch.jl
- Created by: @kchu25
- Version: v0.0.1
- Commit: ac8070200ae8bfccb7afe1a21110417840e5ada6
- Reviewed by: @kchu25
- Reference: https://github.com/kchu25/EpicHyperSketch.jl/commit/ac8070200ae8bfccb7afe1a21110417840e5ada6#commitcomment-168323893
- Description: Parallel enumerations/counting on GPU with fixed space complexity via count-min-sketch
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
- First release.
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd61b95342c6339d8e9385482a161b8e240ef5cd94bfa6056d9040053bf0746b716e9e42db4e6b2cc660f39b371bf065948fd31fa9bd07242f4f6a182412e7044e5af550a05a6c385ff8aee439a2e0e088cc1713d3af054a412ea618b4bdb4f7f9bb0694a2e43d5e60d32f97812c677d31f95722902e4636acbd0231b3f912f88a5a2dcf348cd352877e68cd82d0d32df0bd8e9be4be680f2bde034de76234fe55e5b614b42e66a7cfbcd30f9794f2dfd02 -->

kchu25 commented:
> Thank you for submitting your package. A small nitpick: It seems that the documentation badges at the top of the README are currently broken.
> 
> This package doesn't exactly trigger my "vibe-coding radar", but there do seem to be some Claude-instruction `.md` files that are part of the repo (and probably shouldn't have been committed). Just in case, I would like to point you to a currently ongoing discussion [Should General have a guideline or rule preventing registration of vibe-coded packages?](https://discourse.julialang.org/t/should-general-have-a-guideline-or-rule-preventing-registration-of-vibe-coded-packages/133205). I would trust that this package does not fall into that vibe-coding category, and that you have verified the functionality of the package in detail. It might be helpful if you could fix the code coverage badge at the top of the README.

Hi Michael, thanks for pointing that out! Those markdown files will be removed in the next registration and I'll include the code coverage as well.

JuliaRegistrator committed:
New package: EpicHyperSketch v0.0.1

UUID: c73a71ba-f77a-4450-b81e-a1166e41307d
Repo: https://github.com/kchu25/EpicHyperSketch.jl.git
Tree: d4d819296e6e9e814e6c3d21675b97fcd0d916e2

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
